website/docs/r/policy_tier0_gateway_gre_tunnel.html.markdown

```diff
@@ -0,0 +1,130 @@
+---
+subcategory: "Gateways and Routing"
+layout: "nsxt"
+page_title: "NSXT: nsxt_policy_tier0_gateway_gre_tunnel"
+description: A resource to configure a GRE tunnel on Tier-0 gateway on NSX Policy manager.
+---
+
+# nsxt_policy_tier0_gateway_gre_tunnel
+
+This resource provides a method for the management of a Tier-0 gateway GRE tunnel. Note that edge cluster and in interface must be configured on Tier-0 Gateway in order to configure GRE tunnels on it.
+
+This resource is applicable to NSX Policy Manager.
+
+# Example Usage
+
+```hcl
+data "nsxt_policy_edge_cluster" "EC" {
+  display_name = "EDGECLUSTER1"
+}
+
+data "nsxt_policy_transport_zone" "test" {
+  display_name = "tz-vlan"
+}
+resource "nsxt_policy_vlan_segment" "test" {
+
+  transport_zone_path = data.nsxt_policy_transport_zone.test.path
+  display_name        = "interface_test"
+  vlan_ids            = [11]
+  subnet {
+    cidr = "10.2.2.2/24"
+  }
+}
+data "nsxt_policy_edge_node" "EN" {
+  edge_cluster_path = data.nsxt_policy_edge_cluster.EC.path
+  member_index      = 0
+}
+
+resource "nsxt_policy_tier0_gateway" "test" {
+  display_name = "terraform-acctest-resource-74743"
```

Comment:
pedant nit: For documentation I would use simple names. I think you might have extracted those from an acceptance test run?

Reply:
Exactly... Will replace the name.